src/providers/completer/input.ts

```diff
@@ -19,18 +19,23 @@ export class Input {
     }
 
     private filterIgnoredFiles(files: string[], baseDir: string) : string[] {
-        const excludeGlob = (Object.keys(vscode.workspace.getConfiguration('files', null).get('exclude') || {})).concat(vscode.workspace.getConfiguration('latex-workshop').get('intellisense.file.exclude') || [] ).concat(ignoreFiles)
+        const excludeGlob = Object.keys(vscode.workspace.getConfiguration('files', null).get('exclude') || {})
+            .concat(vscode.workspace.getConfiguration('latex-workshop').get('intellisense.file.exclude') || [])
+            .concat(ignoreFiles)
         let gitIgnoredFiles: string[] = []
         /* Check .gitignore if needed */
         if (vscode.workspace.getConfiguration('search', null).get('useIgnoreFiles')) {
             try {
-                gitIgnoredFiles = (cp.execSync('git check-ignore ' + files.join(' '), {cwd: baseDir})).toString().split('\n')
-            } catch (ex) { }
+                gitIgnoredFiles = cp
+                    .execSync('git check-ignore ' + files.join(' '), { cwd: baseDir })
+                    .toString()
+                    .split('\n')
+            } catch (ex) {}
```

Comment:
I do agree that the reformatting changes of this file improve readability and that they should be integrated at some point but I think you should refrain from introducing such modifications inside a new feature in order to keep a cleaner file history.

Reply:
Oops that was the linter + the `.tslint` file. de30d08bf929461b960d0569c4d99eef955608d7 should make that better.